### PR TITLE
Update changelog prior to next release

### PR DESCRIPTION
**Pull request by namurphy**
This PR revises some changelog entries that have been made since the previous release.

I applied the "No changelog entry needed" label so that the Giles check doesn't give errors related to editing changelog entries for pull requests other than this one.

### CODE REVIEW

File: changelog/2268.doc.rst [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Enabled the ``sphinx.ext.duration`` extension to add links to the source code of an
-object.
+Enabled the `sphinx.ext.duration` extension to show the the times required
```
Comment (namurphy) [inline changelog/2268.doc.rst:1]:
```suggestion
Enabled the `sphinx.ext.duration` extension to show the times required
```

File: docs/contributing/doc_guide.rst [hunk 2]
```diff
@@ -800,7 +800,7 @@ The type specification should not include information about the
    conventions from the `numpydoc style guide
    <https://numpydoc.readthedocs.io/en/latest/format.html#parameters>`__,
    the `matplotlib documentation guide
-   <https://matplotlib.org/stable/devel/documenting_mpl.html#parameter-type-descriptions>`__,
+   <https://matplotlib.org/stable/devel/document.html#parameter-type-descriptions>`__,
```
Comment (namurphy) [inline docs/contributing/doc_guide.rst:803]: link rot strikes again!

File: changelog/2296.trivial.rst [hunk 3]
```diff
@@ -1,2 +1,2 @@
-Add ``ruff`` linter rules that check for ``print`` and ``pprint``, as
+Add |ruff| linter rules that check for :py:`print` and :py:`pprint`, as
```
Comment (namurphy) [inline changelog/2296.trivial.rst:1]:
```suggestion
Add |ruff| linter rules that check for `print` and :py:`pprint`, as
```

File: changelog/2346.trivial.rst [hunk 4]
```diff
@@ -1,2 +1,2 @@
 Enabled |validate_quantities| to accept annotations of the form
-``u.Quantity[u.m]``, where we have run :py:`import astropy.units as u`.
+:py:`u.Quantity[u.m]`, where we have run :py:`import astropy.units as u`.
```
Comment (namurphy) [inline changelog/2346.trivial.rst:2]:
```suggestion
:py:`u.Quantity[u.m]`, where we have previously run :py:`import astropy.units as u`.
```

File: changelog/2262.trivial.rst [hunk 5]
```diff
@@ -1,3 +1,3 @@
-Improved the error message to `~plasmapy.formulary.speeds.Alfven_speed`
+Improved the error message from `~plasmapy.formulary.speeds.Alfven_speed`
```
Comment (namurphy) [inline changelog/2262.trivial.rst:1]:
```suggestion
Improved the error message issued by `~plasmapy.formulary.speeds.Alfven_speed`
```

File: docs/contributing/doc_guide.rst (hunk 2 above)
Comment (ejohnson-96) [reply docs/contributing/doc_guide.rst:803]: Dun dun dunnnnnn

**Review by ejohnson-96 — APPROVED** [review]
A lot of change logs 👍

**Pull request merged**